src/test/unit/rc_controls_unittest.cc

```diff
@@ -262,7 +262,7 @@ class RcControlsAdjustmentsTest : public ::testing::Test {
     controlRateConfig_t controlRateConfig = {
         .thrMid8 = 0,
         .thrExpo8 = 0,
-        .rates_type = RATES_TYPE_BETAFLIGHT,
+        .thrHover8 = 0,
```

Comment:
do not delete `.rates_type = RATES_TYPE_BETAFLIGHT,` 
`thrHover8` should be above rates_type